### DISCUSSION TITLE
Commenting in a PR is non intuitive

### DISCUSSION THREAD

**Discussion by nzjony** [community/community#23359] [New to GitHub]
<p>I generally @ people all the time in a PR. In most cases, the suggestions seem to be ordered in a weird order.</p>
<p>I would like to see it made a bit smarter.</p>
<ul>
<li>When first pressing “@”, the first suggestion is myself, which is a bit weird, I would expect it to be the person I am replying to.</li>
<li>When I want to refine the search and reply to the person, I press “d”, but then it puts the person “datouzan” 3 down, and I have to type “<span class="mention">@dato</span>” to find them. Yes, I could press the down arrow, but to me, if it makes the suggestion as the second person, I would think that pressing ‘d’ would then have them show first, and pressing d is easier than the down arrow.</li>
</ul>
<p>Yes, very minor, but if you are writing a lot of comments each day, the micro annoyances add up.</p>
<p>See the GIF:<br>
<img src="https://user-images.githubusercontent.com/2533428/181076527-ffb69e1d-8b14-4115-8184-206d00f0db6c.gif" alt="out" data-base62-sha1="z7nDP1bVVCIGuOMhM22behp9M5R" width="690" height="398" class="animated"></p>

**Comment by tuves** [comment, marked as answer]
<p>Thanks for this feedback! We’re always working to improve GitHub and the GitHub Support Community; whether it’s minor or not, like you said, with repetition, it can add up. Please submit your feedback via our <a href="https://support.github.com/contact/feedback">feedback form</a> so that our product team can track your request(s).</p>

**Comment by nzjony** [comment]
<p>Thanks <a class="mention" href="/tuves">@tuves</a>, will do!</p>
